gc/base/GCExtensionsBase.cpp

```diff
@@ -180,6 +180,13 @@ MM_GCExtensionsBase::initialize(MM_EnvironmentBase* env)
 		goto failed;
 	}
 
+#if defined(OMRVM_GC_REALTIME)
```

Comment:
Another small thing I just noticed, this code is using two different flags `OMRVM_GC_REALTIME` and `OMR_GC_REALTIME`.  Do you think we could add one of these flags and enable them while this work is ongoing?

Reply:
Typo